aws_lambda_script/main.py

```diff
@@ -0,0 +1,189 @@
+import json
+import csv
+import os
+from typing import Dict, List
+import logging
+from io import StringIO
+import boto3
+
+# Configure logging
+logger = logging.getLogger()
+logger.setLevel(logging.INFO)
+
+# Environment variables
+SOURCE_BUCKET = os.environ.get("SOURCE_BUCKET", "sdp-dev-tech-audit-tool-api")
+SOURCE_KEY = os.environ.get("SOURCE_KEY", "new_project_data.json")
+DESTINATION_BUCKET = os.environ.get("DESTINATION_BUCKET", "sdp-dev-tech-radar")
+DESTINATION_KEY = os.environ.get("DESTINATION_KEY", "onsTechDataAdoption.csv")
+
+
+def get_json_from_s3(bucket: str, key: str) -> Dict:
+    """Retrieve JSON data from S3 bucket"""
```

Comment:
Need more information in your docstrings about arguments and return types. There are some nice vscode extensions to help speed this up. I use [this](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring).

As for docstring style, I use [Google's format](https://google.github.io/styleguide/pyguide.html#38-comments-and-docstrings).

Reply:
Done, thanks for suggestion.